routes/index.ts

```diff
@@ -1,22 +1,23 @@
 import express from "express";
-const router = express.Router();
+import payrollRouter from "./payrolls.routes";
 
-const pickerRouter = require("./pickers.routes");
-router.use(pickerRouter);
+const router = express.Router({ mergeParams: true });
 
-const harvestLogRouter = require("./harvest-logs.routes");
-router.use(harvestLogRouter);
+// const pickerRouter = require("./pickers.routes");
+// router.use(pickerRouter);
 
-const seasonRouter = require("./seasons.routes");
-router.use(seasonRouter);
+// const harvestLogRouter = require("./harvest-logs.routes");
+// router.use(harvestLogRouter);
 
-const productRouter = require("./products.routes");
-router.use(productRouter);
+// const seasonRouter = require("./seasons.routes");
+// router.use(seasonRouter);
 
-const deductionRouter = require("./deductions.routes");
-router.use(deductionRouter);
+// const productRouter = require("./products.routes");
+// router.use(productRouter);
+
+// const deductionRouter = require("./deductions.routes");
+// router.use(deductionRouter);
```

Comment:
I commented these routes out because I changed the require to import. 
If we are okay with this change I can change the rest to import the same way.
@BryanCaldeira @ALFAROSO  @ReinhardtBotha